docs/content/guides/developer/coin/loyalty.mdx

```diff
@@ -8,7 +10,7 @@ The following example demonstrates the creation of a loyalty token that bearers
 
 ## Example
 
-The Loyalty Token example illustrates a loyalty token that is created with the Closed Loop Token standard. If you were to implement this example, the Admin would send `LOYALTY` tokens to the users of your service as a reward for their loyalty. The example creates a `GiftShop` where holders can spend `LOYALTY` tokens to buy `Gift`s.
+The loyalty token example illustrates a loyalty token that is created with the Closed Loop Token standard. If you were to implement this example, the Admin would send `LOYALTY` tokens to the users of your service as a reward for their loyalty. The example creates a `GiftShop` where holders can spend `LOYALTY` tokens to buy `Gift`s.
```

Comment:
I think the idea here was that was going to be the name of the example, but I think you're right that it's not spelled out like that anywhere.